src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -851,8 +857,10 @@ void CountryInstance::_update_military(DefineManager const& define_manager, Unit
 		max_supported_regiment_count += state->get_max_supported_regiments();
 	}
 
-	// TODO - apply country/tech modifiers to supply consumption
-	supply_consumption = 1;
+	supply_consumption = fixed_point_t::_1();
+	if (modifier_effect_cache.get_supply_consumption() != nullptr) {
```

Comment:
Store modifier_effect_cache.get_supply_consumption() in local variable instead of calling it twice.

Reply:
No longer applies